app/device/altimeter.py

```diff
@@ -0,0 +1,73 @@
+from app.utils.i2c import I2C
+from time import sleep
+
+MPL3115A2_ADDRESS = 0x60
+STATUS = 0x00
+OUT_P_MSB = 0x01
+OUT_P_DELTA_CSB = 0x08
+OUT_P_DELTA_MSB = 0X07
+OUT_T_MSB = 0x04
+WHOAMI = 0x0C
+CTRL_REG1 = 0x26
+PT_DATA_CFG = 0x13
+BAR_IN_MSB = 0x14
+
+class Altimeter (object):
+
+    def __init__(self):
+
+        while(True):
```

Comment:
we should not use while true, instead set a retry limit and then raise an exception if it still fails